torch_geometric/explainability/algo/captumexplainer.py

```diff
@@ -0,0 +1,147 @@
+import torch
+
+from torch_geometric.explainability.algo.base import ExplainerAlgorithm
+from torch_geometric.explainability.explanations import Explanation
+from torch_geometric.explainability.utils import CaptumModel
+
+
+class CaptumExplainer(ExplainerAlgorithm):
+    """Wrapper for captum explainers.
+
+    Wraper for captum explainers.
+
+    Note:
+        This algorithm expect the model to be callable with the following
+        signature: model(x, edge_index, **kwargs).
+
+
+    Args:
+        method (str, optional): name of the captum explainer.
+        Defaults to "Saliency".
+    """
+    def __init__(self, method: str = "Saliency") -> None:
+
+        super().__init__()
+        self.method = method
+
+    def explain(
+        self,
+        x: torch.Tensor,
+        edge_index: torch.Tensor,
+        target: torch.Tensor,
+        model: CaptumModel,
+        mask_type: str = "node",
+        **kwargs,
+    ) -> Explanation:
+
```

Comment:
we could probably add docstrings for various arguments (also specify what kwargs could include).